packages/gestalt/src/contexts/DeviceTypeProvider.js

```diff
@@ -0,0 +1,31 @@
+// @flow strict
+import { type Context, type Node, createContext, useContext } from 'react';
+
+const defaultDeviceType = 'desktop';
+
+type DeviceType = 'desktop' | 'phone' | 'tablet';
```

Comment:
Currently we show/hide content through breakpoints (e.g. `smDisplay` / `mdDisplay` and `lgDisplay`) instead of device type. It would be good to clarify when we should use what.